packages/react-native-lockdown/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@lavamoat/react-native-lockdown",
+  "version": "0.0.1-beta.0",
+  "description": "LavaMoat React Native lockdown provides Hardened JavaScript in React Native apps via Metro",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LavaMoat/lavamoat.git",
+    "directory": "packages/react-native-lockdown"
+  },
+  "homepage": "https://github.com/LavaMoat/lavamoat#readme",
+  "bugs": {
+    "url": "https://github.com/LavaMoat/lavamoat/issues"
+  },
+  "author": "LavaMoat Team",
+  "license": "MIT",
+  "engines": {
+    "node": ">18.0.0"
+  },
+  "main": "./src/index.js",
+  "exports": {
+    ".": "./src/index.js",
+    "./repair": "./src/repair.js"
+  },
+  "types": "./types/src/types.d.ts",
+  "files": [
+    "CHANGELOG.md",
+    "src",
+    "types",
+    "!*.tsbuildinfo"
+  ],
+  "scripts": {
+    "test": "npm run test:run",
+    "test:run": "ava",
+    "test:update-snapshots": "ava -u"
+  },
+  "peerDependencies": {
+    "@react-native/js-polyfills": "*"
+  },
+  "dependencies": {
+    "ses": "^1.12.0"
+  },
+  "devDependencies": {
+    "@react-native/js-polyfills": "^0.79.1"
```

```suggestion
    "@react-native/js-polyfills": "0.79.2"
```

The latest is `0.79.2` as of this writing. you'll need to re-run `npm i` and update the lockfile